tests/unit/build-waiter-test.ts

```diff
@@ -11,44 +12,44 @@ module('test-waiter', function(hooks) {
 
   test('test waiter can be instantiated with a name', function(assert) {
     let name = 'my-waiter';
-    let waiter = new TestWaiter(name);
+    let waiter = buildWaiter(name);
 
     assert.equal(waiter.name, name);
   });
 
   test('test waiters return a token from beginAsync when no token provided', function(assert) {
-    let waiter = new TestWaiter('my-waiter');
+    let waiter = buildWaiter('my-waiter');
 
     let token = waiter.beginAsync();
 
     assert.ok(token instanceof Token, 'A token was returned from beginAsync');
   });
 
   test('test waiters return a truthy token from beginAsync when no token provided', function(assert) {
-    let waiter = new TestWaiter('my-waiter');
+    let waiter = buildWaiter('my-waiter');
 
     let token = waiter.beginAsync();
 
     assert.ok(token, 'A token was returned from beginAsync and is truthy');
   });
 
   test('test waiters automatically register when beginAsync is invoked when no token provided', function(assert) {
-    let waiter = new TestWaiter('my-waiter');
+    let waiter = buildWaiter('my-waiter');
 
     let token = waiter.beginAsync();
 
     let registeredWaiters = getWaiters();
 
     assert.equal(registeredWaiters[0], waiter, 'The waiter is registered');
     assert.deepEqual(
-      (<TestWaiter>registeredWaiters[0]).items.keys().next().value,
+      (<TestWaiterImpl>registeredWaiters[0]).items.keys().next().value,
```

Comment:
Hmmm, I'd rather cast than export the class just for this purpose...

Reply:
Cast to what? The interface doesn't have an items prop.

Reply:
```ts
let waiter = registeredWaiters[0] as any;
assert.deepEqual(waiter.items.keys().next().value, whateverHere)
```

Reply:
We should refactor these tests to leverage the debug info apis instead of accessing `.items` privately, once we do that the nastiness of the any cast can go away...

Reply:
In this case this test is explicitly looking for a token's presence. We don't expose the token via our debug info, so that won't work. I've resigned to cast to `any` for now until we can clean this up later.